### PR TITLE
GLTFExporter: improve error message when exporting unsupported light

### PR DESCRIPTION
**Pull request by garyo**

### CODE REVIEW

File: examples/js/exporters/GLTFExporter.js [hunk 1]
```diff
@@ -1754,7 +1754,7 @@ THREE.GLTFExporter.prototype = {
 
 			} else if ( object.isLight ) {
 
-				console.warn( 'THREE.GLTFExporter: Only directional, point, and spot lights are supported.' );
+				console.warn( `THREE.GLTFExporter: light ${object.name} of type ${object.constructor.name}: Only directional, point, and spot lights are supported.` );
```
Comment (mrdoob) [inline examples/js/exporters/GLTFExporter.js]: We usually do this instead:
```js
console.warn( 'THREE.GLTFExporter: Only directional, point, and spot lights are supported.', object );
```

Comment (garyo) [reply examples/js/exporters/GLTFExporter.js]: Would that show the type though? That's what a user would need to search for.

Comment (Mugen87) [reply examples/js/exporters/GLTFExporter.js]: The warning looks in Chrome like so:

image

I think it's okay like that. Users can still debug the object for more details.

**Comment by garyo** [comment]
done.

**Review by donmccurdy — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**